collectors/ebpf.plugin/ebpf.d/hardirq.conf

```diff
@@ -3,6 +3,9 @@
 #  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
 #            new charts for the return of these functions, such as errors.
 #
-#[global]
+# The `life time` defines the time length a thread will run when is enabled by a function.
```

Comment:
Ah, I hate to do this, but I think it should be `....will run when it is enabled....`

Reply:
> The "life time" parameter specifies how long the thread will run if enabled through functions.

I am not sure "the time length" fits here.

Also, consider renaming it to `lifetime`

Reply:
> Ah, I hate to do this, but I think it should be ....will run when it is enabled....

In Latin languages we almost never use pronouns :facepalm:, thank you! 

@ilyam8 I updated for `lifetime`.